locale/en/blog/vulnerability/september-2017-path-validation.md

```diff
@@ -28,7 +29,7 @@ attacker may be able to access file system paths other than those intended.
 This problem was resolved within Node.js by partially reverting
 https://github.com/nodejs/node/commit/b98e8d995efb426bbdee56ce503017bdcbbc6332.
 
-A CVE will be requested and the number will be posted once available.
+A CVE has been assigned as [CVE-2017-14849](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-14849)
```

Comment:
Micro-nit, feel free to ignore, but maybe just keep it simple and non-passive:

```
This issue has been assigned [CVE-2017-14849](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-14849).
```

Reply:
-1 in this case, two consecutive sentences starting with "This …"

Reply:
I feel that's less awkward than "has been assigned as", but maybe we can avoid both problems:

```
[CVE-2017-14849](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-14849) has been assigned for this vulnerability.
```